internal/util-logging/src/main/scala/sbt/internal/util/codec/JValueFormats.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package sbt
+package internal
+package util.codec
+
+import sjsonnew.{ JsonWriter => JW, JsonReader => JR, JsonFormat => JF, _ }
+import scala.json.ast.unsafe._
+
+trait JValueFormats { self: sjsonnew.BasicJsonProtocol =>
```

Comment:
eventually I think this should move to sjson-new, in the scala-json-ast support jar.